examples/3d/shadow_biases.rs

```diff
@@ -66,15 +66,7 @@ fn setup(
     commands.spawn(DirectionalLightBundle {
         directional_light: DirectionalLight {
             illuminance: 100000.0,
-            shadow_projection: OrthographicProjection {
-                left: -0.35,
-                right: 500.35,
-                bottom: -0.1,
-                top: 5.0,
-                near: -5.0,
-                far: 5.0,
-                ..default()
-            },
+            shadow_projection: OrthographicProjection::new().clipping_plane(-5.0, 5.0),
```

Comment:
It feels like the original example with its `-0.35` and `500.35` was carefully crafted, and I very much doubt that new version is equivalent. @bevyengine/rendering-team can someone shed some light here why that original projection had very specific values on _e.g._ `left` or `right`?

Reply:
I assumed this was a misunderstanding of the `left`, `right`, etc. fields (which would be ironic) since I don't believe they actually had any affect (`ScalingMode` defaults to `WindowSize`, which doesn't take them into account), but I could be wrong. I didn't notice any changes in the example, but maybe a previous change to `OrthographicProjection` caused an issue here.

Reply:
Also, I've put off testing examples extensively until changes are finalized. I'm sure I've broken some other examples.

Reply:
Fair enough. Let's see what the rendering team has to say, if any.